### DISCUSSION TITLE
InfinIdea

### DISCUSSION THREAD

**Discussion by pierrbt** [Mysterious-Developers/InfinIdea#33] [General]
Vous pouvez commencer à discuter dans le salon général du repo de InfinIdea

**Comment by olarno** [comment]
Bonjour, comment peut-on se rendre utile pour le projet ? 
Voir une pre-prod ? 

enfin tout simplement comment contribuer ce projet ;)

**Comment by pierrbt** [comment]
@olarno 
> Bonjour, comment peut-on se rendre utile pour le projet ? 
> Voir une pre-prod ? 
> 
> enfin tout simplement comment contribuer ce projet ;) 
> 

Bienvenue sur le projet !
Nous sommes très content que vous soyez partant pour tester le site. Il y a toutes les informations dans le README concernant les contributions je te laisse aller lire. Pour accéder au site tu peux aller sur [infinidea.veagle.fr](https://infinidea.veagle.fr).
Si tu as d'autres questions n'hésite surtout pas à les poser ici ou dans les Issues.
Pierre

**Comment by pierrbt** [comment]
@olarno 
Et surtout n'hésite pas, nous n'avons pas encore beaucoup de contributeurs donc si tu as un conseil, une amélioration ou que tu trouves une faille ou un bug nous sommes très ouverts. Bonne journée.
L'équipe Veagle
